samples/bluetooth/broadcast_audio_assistant/prj.conf

```diff
@@ -11,4 +11,7 @@ CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=191
 CONFIG_BT_TINYCRYPT_ECC=y
 
 CONFIG_BT_EXT_ADV=y
+CONFIG_BT_BAP_BASS_MAX_SUBGROUPS=2
 CONFIG_BT_BAP_BROADCAST_ASSISTANT=y
+CONFIG_BT_AUDIO_CODEC_CFG_MAX_METADATA_SIZE=255
+CONFIG_SELECT_SOURCE_NAME="Broadcast Audio Source"
```

Comment:
It doesn't make sense to set `CONFIG_SELECT_SOURCE_NAME` to the non-default value in this file IMO. 

Suggest to remove this and keep the default to "" so it connects to the first again (typical behavior in our samples)